tensorboard/plugins/pr_curve/summary.py

```diff
@@ -31,24 +31,19 @@ def op(
     labels,
     predictions,
     num_thresholds=200,
-    weights=None,
-    num_classes=1,
+    weight=1.0,
     display_name=None,
-    description=None):
-  """Computes multi-class PR summaries for a list of thresholds in `[0, 1]`.
+    description=''):
```

Comment:
Correct; this is what I meant by "consistent mutable-default-value initialization." Default arguments are values and not expressions (cf. JavaScript), so they are evaluated at function definition time and not function invocation time. Writing `my_list=[]` is nearly always wrong, and is discouraged by Google style

> Default arguments are evaluated once at module load time. This may cause problems if the argument is a mutable object such as a list or a dictionary. If the function modifies the object (e.g., by appending an item to a list), the default value is modified.

; I recently also learned that you can write
```py
>>> def my_function(foo=[]):
...     return foo
... 
>>> my_function.func_defaults[0]
[]
>>> my_function.func_defaults = (['banana'], )
>>> my_function()
['banana']
```
which makes them even more dangerous.